src/cascadia/TerminalControl/SearchBoxControl.cpp

```diff
@@ -4,6 +4,7 @@
 #include "pch.h"
 #include "SearchBoxControl.h"
 #include "SearchBoxControl.g.cpp"
+#include <LibraryResources.h>

```

Comment:
You should only need the `resw` one for this, because it will automatically populate the attached property when it loads the resource tree. If that isn't working, ... then that sucks. Let me know.

Reply:
Yeah it works. This was more because I have trust issues; like, this would work in the event the localization team doesn't read "This should match 'SearchBox_TextBox.PlaceholderText'" haha